aspnetcore/blazor/security/webassembly/standalone-with-azure-active-directory-b2c.md

```diff
@@ -67,12 +91,18 @@ dotnet new blazorwasm -au IndividualB2C --aad-b2c-instance "{AAD B2C INSTANCE}"
 The output location specified with the `-o|--output` option creates a project folder if it doesn't exist and becomes part of the app's name.
 
 > [!NOTE]
-> In the Azure portal, the app's **Authentication** > **Platform configurations** > **Web** > **Redirect URI** is configured for port 5001 for apps that run on the Kestrel server with default settings.
+> In the Azure portal, the app's platform configuration **Redirect URI** is configured for port 5001 for apps that run on the Kestrel server with default settings.
 >
 > If the app is run on a random IIS Express port, the port for the app can be found in the app's properties in the **Debug** panel.
 >
 > If the port wasn't configured earlier with the app's known port, return to the app's registration in the Azure portal and update the redirect URI with the correct port.
 
+::: moniker range=">= aspnetcore-5.0"
+
+[!INCLUDE[](~/includes/blazor-security/additional-scopes-standalone.md)]
```

Comment:
AAD and B2C needs different scopes so I think we'll need different versions of these includes.

AAD needs the `User.Read` scope and `B2C` uses the `openid` ones.

Also, the config shouldn't be needed for hosted apps.

Reply:
I only tested standalone with Microsoft Accounts this morning. The scopes are only `openid` and `offline_access` for that scenario. I incorrectly assumed that the AAD and AAD B2C scenarios would match. I'll get this resolved on the next commit.

> Also, the config shouldn't be needed for hosted apps.

Do you mean for the *Server* app of the hosted solution? I agree, but let's focus on the two hosted scenarios on another PR, perhaps tomorrow.

Reply:
Ah! ... yes ... I see what you mean now. Good :eye:. I really should have taken a little more time with it this morning.